source/includes/_plugin_extending_pages.md

```diff
@@ -36,8 +68,9 @@ class PageSubscriber extends CommonSubscriber
     public function onPageBuild(PageBuilderEvent $event)
     {
         // Add page tokens
-        $content = $this->templating->render('HelloWorldBundle:SubscribedEvents\PageToken:token.html.php');
-        $event->addTokenSection('helloworld.token', 'plugin.helloworld.header', $content);
```

Comment:
This function [was deleted](https://github.com/mautic/mautic/blob/staging/UPGRADE-3.0.md#corebundle) in Mautic 3.0.0, without any hint as to what should be used instead.

Reply:
@dennisameling this method was removed over 5 years ago https://github.com/mautic-inc/mautic-cloud/commit/5ac67e5293eadfa551f21023a6345196fa6c861c#diff-ecfcb8a8d5446acbe2a7dfe89af7f3eb6126f89f4b939e4a522461b0a25057df

Look at for example https://github.com/mautic/mautic/blob/staging/app/bundles/AssetBundle/EventListener/BuilderSubscriber.php which is a good example how to work with tokens.